src/common/ci/github/commentOnPR.ts

```diff
@@ -51,7 +56,7 @@ export const commentOnPR = async (comment: string, signOff: string) => {
       });
     }
   } catch (error) {
-    logger.error(`Failed to comment on PR: ${error}`);
+    logger.error(`Failed to comment on PR: ${error as string}`);
```

Comment:
[Note] I'd like to create a follow up ticket to remove unnecessary casting from the repo as I think we have some type safety issues atm 🤔 This is a temporary fix

Reply:
You should be able to use JSON.stringify for this error object.

Reply:
Yeah, I just went for a hacky universal fix- I was gonna look into what best practice was for this specific scenario in a separate casting ticket

Reply:
There is a fair chance the best fix will be JSON.stringify though 😄